### ISSUE TITLE
Stack Exchange pages seem to show only the question or one answer in Reader View

### ISSUE THREAD

**Issue by svick** [mozilla/readability#232]
On Stack Exchange sites, I think that Reader View should display the question and all the answers, but that's not what I see.

For example, on http://programmers.stackexchange.com/questions/283315/when-asynchronous-tasks-make-a-bad-ux, Reader View displays only the question, not the answers.

On the other hand, on http://english.stackexchange.com/questions/250245/a-hypernym-for-insects-worms-and-the-like, only the third answer is displayed.

**Comment by tjanson** [comment]
Yeah, I’m surprised StackExchange isn’t supported – or at least fully disabled. It’s currently not helpful at all.
Consider  [this page](http://stackoverflow.com/questions/30615447/how-to-disable-firefoxs-reader-view-from-my-website) ([full page screenshot](http://i.imgur.com/wxcBVPu.png)):

> ![](http://i.imgur.com/p94UZd6.png)

Gets turned into this, a 6000px wide code snippet from an answer:

> ![](http://i.imgur.com/abCPYkV.png)

**Comment by netjeff** [comment]
FYI, I tried the two links from @svick's original report:

- http://programmers.stackexchange.com/questions/283315/when-asynchronous-tasks-make-a-bad-ux
- http://english.stackexchange.com/questions/250245/a-hypernym-for-insects-worms-and-the-like

Reader View in FF 72.0.1 on Win10 shows both question and answers.
